portfolio/src/main/webapp/projects/adventure.html

```diff
@@ -0,0 +1,45 @@
+<!-- Works in tendem with image_template.html and button_template.html-->
+<!-- TOKENS: NAME, IMAGES, GITHUB_BUTTON-->
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Poh Lin Wei">
+    <meta name="description" content="project Adventure ">
+    <meta name="viewport" content="width=device-width,initial-scale=1">
+    <title>Project: Adventure</title>
+    <link rel="stylesheet" href="style.css">
+  </head>
+  <body>
+    <nav>
+      <a href="../index.html#works">Back to Main</a>
+    </nav>
+    <header>
+      <div id="logo"/>
+    </header>
+    <main>
+      <h1 class="project-name">Adventure</h1>
+      <picture>
+        <source media="(max-width: 399px), (orientation: portrait)" srcset="adventure/0_mobile.png" size="400px">  
+        <source media="(min-width: 400px)" srcset="adventure/0.png" size="400px">
+        <img class="screenshot" src="adventure/0_mobile.png"" alt="Screenshot of Adventure">
+      </picture>      <picture>
```

Comment:
I think there are some formatting issues here

Reply:
Thanks for pointing this out!